tools/import_vtree_from_freebsd.sh

```diff
@@ -16,8 +16,7 @@ git diff vtree.h | git apply -R > /dev/null 2>&1 || true
 GR=f6e54eb360a78856dcde930a00d9b2b3627309ab
```

Comment:
The commit hash currently tracked by `$GR` does not match our current import of `vtree.h`.